explainaboard/info.py

```diff
@@ -87,6 +87,8 @@ class SysOutputInfo:
     metric_names: Optional[list[str]] = None
     reload_stat: bool = True
     is_print_case: bool = True
+    is_print_confidence_interval: bool = False
```

Comment:
`is_print_confidence_interval` has been replaced by `conf_value`. Also, I think confidence intervals should be on by default, no reason to have them off really.

Reply:
>"user_defined_metadata_configs is very verbose. Maybe just metadata is fine"

Good point. So far I used it to make a consistent naming with `user_defined_features_configs`, We can re-organize them together later. Also some new and unexpected `xyz-config` may come in the future.

> "is_print_confidence_interval has been replaced"

OK. removed!

>"Should analyzers be called visualizers"

I have considered it, so far I reuse the folder `analyzer` that Chihao has set up.